app/views/alaveteli_pro/info_request_batches/_message_preview.html.erb

```diff
@@ -27,7 +27,18 @@
               'request to make a new but similar batch of requests.',
               date: simple_date(@info_request_batch.existing_batch.created_at),
               existing_batch: info_request_batch_path(@info_request_batch.existing_batch)) %>
+
+        <%= _('As a Pro user, you can choose to ignore the existing batch by ' \
+              'checking the box below and send your requests. Please ' \
+              'remember you are still bound by our <a href="{{url}}">House ' \
+              'Rules on making responsible requests</a>.',
+              url: help_requesting_path(anchor: 'responsible')) %>
```

Comment:
> We may want to change the `InfoRequestBatch.find_existing` to return an existing batch if any of the new batch's public bodies were included in the previous batch.

I agree, I think we should do this, which would then allow the full `errorExplanation` text to read:

> **You have already sent this request** to some authorities in this batch on 13 June 2022.
>
> You can [view the existing batch](), [edit your request](), or ignore this warning by checking the box below.
>
> Please remember that our [terms and conditions]() include **making responsible requests**.

`help_requesting_path(anchor: 'responsible')` isn't a [required section](https://alaveteli.org/docs/customising/themes/#customising-the-help-pages), but `/pro/pages/legal` is when pro is enabled.

Would ideally make each line a `<p>` – should work, but no big deal if it introduces lots of formatting hassle.